packages/cli/src/commands/index.ts

```diff
@@ -30,4 +31,5 @@ export const commands: CommandModule<any, any>[] = [
   devContracts,
   abiTs,
   verify,
+  init,
```

Comment:
prob need a changeset for this (and in a future PR, can edit/expand the changeset to include new behavior)